src/components/Filters/Filters.tsx

```diff
@@ -1,22 +1,39 @@
-import React, { useState, useRef } from 'react';
+import React, { useState } from 'react';
 import styles from './Filters.module.css';
 import Button from '../Button';
-import { CheckboxField, FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
+import { FiltersCheckboxField } from '../FiltersCheckboxField/FiltersCheckboxField';
+import { FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
 import Icon from '../Icon';
 
 export type Props = {
   /**
-   * Checkbox fields that will be displayed in the filters drawer.
+   * Text to be placed in the button that activates the Filters Drawer
    */
-  checkboxFields: CheckboxField[];
+  buttonText?: string;
+  /**
+   * Button status variant that indicates if filters had been selected.
+   */
+  buttonStatusVariant: 'neutral' | 'brand';
```

Comment:
I found this logic kinda confusing. What if instead of asking for the button status, we instead asked whether any filters have been applied? I think that would be easier to understand and one less bit of logic the dev has to write (ie `buttonStatusVariant={hasFilters ? 'brand' : 'neutral'}`).

Reply:
Mmmmmm great suggestion will change to `hasFilters` prop